accounts/models.py

```diff
@@ -0,0 +1,72 @@
+from django.db import models
+from django.contrib.auth.models import (BaseUserManager, AbstractBaseUser)
+
+
+class MyUserManager(BaseUserManager):
+    use_in_migrations = True
+
+    def _create_user(self, email, password, **extra_fields):
+        """
+        Creates and saves a User with the given email and password.
+        """
+        if not email:
+            raise ValueError('The given email must be set')
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_user(self, email, password=None, **extra_fields):
+        extra_fields.setdefault('is_admin', False)
+        return self._create_user(email, password, **extra_fields)
+
+    def create_superuser(self, email, password=None):
+        """
+        Creates and saves a superuser with the given email, password.
+        """
+        user = self.create_user(
+            email,
+            password=password,
+        )
+        user.is_admin = True
+        user.save(using=self._db)
+        return user
+
+
+class MyUser(AbstractBaseUser):
+    email = models.EmailField(verbose_name='email address', max_length=255, unique=True)
+    first_name = models.CharField(max_length=50, blank=True)
+    last_name = models.CharField(max_length=50, blank=True)
+    date_joined = models.DateTimeField(verbose_name='date joined', auto_now_add=True)
+    is_active = models.BooleanField(default=True)
+    is_admin = models.BooleanField(default=False)
+    avatar = models.ImageField(upload_to='avatars/', null=True, blank=True)
+    order = models.ForeignKey('order.Order', on_delete=models.SET_NULL, null=True, blank=True)
+
+    objects = MyUserManager()
+
+    USERNAME_FIELD = 'email'
+
+    class Meta:
+        verbose_name = 'user',
+        verbose_name_plural = 'users'
+
+    def __str__(self):
+        return self.email
+
+    def has_perm(self, perm, obj=None):
+        "Does the user have a specific permission?"
```

Comment:
```suggestion
        """Does the user have a specific permission?"""
```
docstrings should be written this way

Reply:
Исправил